client/python/ovmsclient/samples/README.md

```diff
@@ -219,24 +236,42 @@ usage: grpc_predict_binary_vehicle_detection.py [-h] [--images_dir IMAGES_DIR]
 
 optional arguments:
   -h, --help            show this help message and exit
-  --images_dir IMAGES_DIR Path to a directory with images in JPG or PNG format
-  --service_url SERVICE_URL Specify url to grpc service. default:localhost:9000
-  --model_name MODEL_NAME Model name to query. default: vehicle-detection
-  --model_version MODEL_VERSION Model version to query. default: latest available
-  --output_dir OUTPUT_DIR Path to store output.
-  --timeout TIMEOUT Request timeout. default: 10.0
+  --images_dir IMAGES_DIR
+                        Path to a directory with images in JPG or PNG format
+  --service_url SERVICE_URL
+                        Specify url to grpc service. default:localhost:9000
+  --model_name MODEL_NAME
+                        Model name to query. default: vehicle-detection
+  --model_version MODEL_VERSION
+                        Model version to query. default: latest available
+  --output_dir OUTPUT_DIR
+                        Path to store output.
+  --timeout TIMEOUT     Request timeout. default: 10.0
 ```
 
 - Usage example
 
 ```bash
-python grpc_predict_binary_vehicle_detection.py --images_dir ../../../../demos/common/static/images/cars/ --output_dir ./output
+python grpc_predict_binary_vehicle_detection.py --images_dir ../../../../demos/common/static/images/cars/ --output_dir ./output --service_url localhost:9000
 Making directory for output: ./output
 Detection results in file:  ./output/road1.jpg
 ```
 
 ## HTTP Client Examples <a name="http-api"></a>
 
+Download [Resnet50-tf Model](https://docs.openvino.ai/2022.1/omz_models_model_resnet_50_tf.html) and convert it into Intermediate Representation format:
+```bash
+mkdir models
+docker run -u $(id -u):$(id -g) -v ${PWD}/models:/models openvino/ubuntu18_dev:latest omz_downloader --name resnet-50-tf --output_dir /models
+docker run -u $(id -u):$(id -g) -v ${PWD}/models:/models:rw openvino/ubuntu18_dev:latest omz_converter --name resnet-50-tf --download_dir /models --output_dir /models --precisions FP32
+mv ${PWD}/models/public/resnet-50-tf/FP32 ${PWD}/models/public/resnet-50-tf/1
+```
+
+OVMS can be started using a command:
+```bash
+docker run -d --rm -v ${PWD}/models/public/resnet-50-tf:/models/public/resnet-50-tf -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path /models/public/resnet-50-tf --port 9000 --rest_port 8000 
+```
+
```

Comment:
I followed convention that was used for Vehicle detection model part, but you're right there is no need to download models for the second time. 
However I do need to start ovms container again since test follows every command in order from the document. So if I don't "python http_get_model_status.py --model_name resnet" command will fail.

If I'm removing models part from here I would propose to also remove downloading models for Vehicle detection model for the second time.

Reply:
I see we also do that for Resnet. In that case let's keep all these instructions. At least for now...

Reply:
Removed additional model downloads as we discussed.